src/azure-cli-core/azure/cli/core/commands/template_create.py

```diff
@@ -90,7 +90,7 @@ def get_folded_parameter_validator(
         raise CLIError('Cannot use default_none=True if allow_none=False')
 
     # construct the validator
-    def validator(namespace):
+    def validator(cmd, namespace):  # pylint: disable=unused-argument
```

Comment:
Is this change needed, as I don't see any usages  in the body.  Or at the line 113, we should the update `namespace.cmd.cli_ctx`?

Reply:
It is necessary as I tried to remove it do get rid of the pylint warning and there were a billion errors.  I will fix up the namespace.cmd.cli_ctx reference. Thanks for catching it!